rest-api-spec/src/main/resources/rest-api-spec/test/index/110_constant_keyword.yml

```diff
@@ -0,0 +1,70 @@
+---
+setup:
+  - do:
+      indices.create:
+        index: test
+        body:
+          mappings:
+            properties:
+              genre:
+                type : "constant_keyword"
+                "value" : "1"
+
+  - do:
+      index:
+        index: test
+        id: 1
+        body: {
+          "genre": "1"
+        }
+
+  - do:
+      index:
+        index: test
+        id: 2
+        body: {
+          "genre": 1
+        }
+
+  - do:
+      indices.refresh:
+        index: test
+
+---
+# Delete Index when connection is teardown
+teardown:
+  - do:
+      indices.delete:
+        index: test
+
+---
+"Mappings":
+  - skip:
+      version: " - 2.13.99"
```

Comment:
The test failure relates to `org.opensearch.backwards.MixedClusterClientYamlTestSuiteIT`, the test case fails on a old version cluster definitely because we haven't backported this PR to 2.x, so we need to change the skip version to `- 2.99.99`, after this PR is merged into main branch and backported to 2.x branch, we need to open another PR to change the skip version to `2.15.99`, and if we want backport this PR to 2.15 and 2.14 branch(I think we should do), then we need to change the skip version to `2.13.99` in the new PR after all the backport completes.

See https://github.com/opensearch-project/OpenSearch/pull/14793.

Reply:
Thank you very much for pointing out the key of the problem. I have always been curious why CI always throws the same exception.